middle_end/flambda2/parser/fexpr_to_flambda.ml

```diff
@@ -288,7 +288,7 @@ let rec subkind : Fexpr.subkind -> Flambda_kind.With_subkind.Subkind.t =
 and value_kind_with_subkind :
     Fexpr.kind_with_subkind -> Flambda_kind.With_subkind.t = function
   | Value sk ->
-    Flambda_kind.With_subkind.create Flambda_kind.value (sk |> subkind)
+    Flambda_kind.With_subkind.create Flambda_kind.value (sk |> subkind) Nullable
```

Comment:
This causes failing fexpr tests, because we fail to unbox floats (and other related things) when we only have the kind information to unbox.